README.md

```diff
@@ -15,18 +15,23 @@ After installation, the `proxygen` executable is available. Typing `proxygen` di
 
 The CLI requires client credentials, which can be obtained from the `platforms-api-producer-support` Slack channel.
 
-All users should also have individual credentials. `proxygen-cli` needs to know about them.
-You can setup the credentials in `proxygen-cli` using:
+The Proxygen CLI client credentials are setup now as part of proxygen cli package.
```

Comment:
I think would be better to just explain it like it is currently. Rarther than saying 'Now its like this' could we reword it to be more 'It is like this'

Reply:
Sure Alex I will make changes and Update the PR